src/Elmish.WPF/Binding.fs

```diff
@@ -498,7 +498,9 @@ type Binding private () =
        itemEquals: 'b -> 'b -> bool,
        getId: 'b -> 'id)
       : string -> Binding<'model, 'msg> =
-    BindingData.OneWaySeqLazy.create get equals map itemEquals getId
```

Comment:
I should a add this (beta-breaking) change to the release notes

Reply:
Was this the only place this was used? I thought there were more...

Reply:
Oh, that is not part of the public API.  Instead, I should expose something like this to the public API.